### ISSUE TITLE
SELinux policy for binaries

### ISSUE THREAD

**Issue by aakselrod** [NYDIG/lndsigner#26]
Best practices for Vault includes setting an [SELinux](https://www.hashicorp.com/blog/hardening-hashicorp-vault-with-selinux) policy. We should develop a policy for:

- [ ] the vault plugin
- [ ] the sidecar binary
- [ ] potentially (depending on #10), the policy enforcement binary

**Comment by aakselrod** [comment]
Duplicate of https://github.com/NYDIG-OSS/lndsigner/issues/20
